Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml

```diff
@@ -1848,7 +2170,835 @@ script:
     - contextPath: AzureSentinel.ThreatIndicator.Types
       description: The threat types of the indicator.
       type: Unknown
-  dockerimage: demisto/crypto:1.0.0.47970
+  - arguments:
+    - default: false
+      description: Date string representing the local time. The incident is only returned if it was modified after the last update time.
+      name: lastUpdate
+      required: true
+    description: Gets the list of incidents and detections that were modified since the last update time. This method is used for debugging purposes. The get-modified-remote-data command is used as part of the Mirroring feature that was introduced in Cortex XSOAR version 6.1.
```

Comment:
Is it necessary to say " that was introduced in Cortex XSOAR version 6.1"?
We are in version 6.10 with 6.11 due to be released next month.
6.1 was a long time ago.

Reply:
I will remove it